conda_smithy/lint_recipe.py

```diff
@@ -59,9 +58,9 @@ def get_section(parent, name, lints):
 
 def get_list_section(parent, name, lints, allow_single=False):
     section = parent.get(name, [])
-    if allow_single and isinstance(section, dict):
+    if allow_single and isinstance(section, Mapping):
         return [section]
-    elif isinstance(section, list):
+    elif isinstance(section, Sequence):
```

Comment:
Do we care if this could be a string?

Reply:
Blegh. Maybe...the linter will probably crash if someone does `source: "haha"` then.

Reply:
maybe use this with not `isinstance(section, str)`? https://stackoverflow.com/questions/1835018/how-to-check-if-an-object-is-a-list-or-tuple-but-not-string/1835259